CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 2.12 beta v1 (August 2019)
+- Shares rearchitecture
+- UI improvements, including:
+  + Private link accessible when share API is disabled
+- Bug fixes, including:
+  + Fix images not detected in Android 9 gallery after being downloaded
+  + Fix crash when notifying upload result
```

Comment:
This one was already included in 2.11.1 (you can see it just below)

Reply:
Done